src/index.js

```diff
@@ -747,6 +747,39 @@ walletRouter.post(
   }
 );
 
+/**
+ * GET request to obtain adress information
+ * For the docs, see api-docs.js
+ */
+ walletRouter.get(
+  '/address-info',
+  query('address').isString(),
+  query('token').isString().optional(),
+  (req, res) => {
+    try {
+      // Query parameters validation
+      const validationResult = parametersValidation(req);
+      if (!validationResult.success) {
+        return res.status(400).json(validationResult);
+      }
+
+      const wallet = req.wallet;
+      const { address, token } = matchedData(req, { locations: ['query'] });
+      const result = wallet.getAddressInfo(address, { token });
```

Comment:
I feel this is the place where the exception might happen. So, we should keep the try-catch block just around it.

Reply:
Done, thanks.